### DISCUSSION TITLE
Unable to boot into GUI and issues with webradio and update

### DISCUSSION THREAD

**Discussion by GeorgeAmb** [rern/rAudio#1197] [General]
I am very new to this and am trying to get a working webradio.
Downloaded Release i20221117 for my Raspberry 4 and wrote the image. (Also tried on a PI 3) There is no additional hardware in my PI's
When I start the device I see the rAudio-1 Logo but nothing happens for hours.
When I hit Ctrl+Alt+F2 i can login with root and then I have to enter Startx.
When I do that booting continues and the GUI is started.
Rebooting the device does the same. So idea how to configure autostart? (Could someone explain for a noob?)
I then tried to download an older release (20220904).
This Image seems to boot fine automatically :)
However as soon as I install the "Webradio plugin" it will fail and say "Error in JSON at line 1871) which doesn't seem right.

I also noticed there is a 20230117 version but there seems to be no image for that. And in the 20221117 image it's not possible to press "update", it says one needs to download the newer image.
How does one get the update? Is it possible from the commandline?
Anyway, maybe someone can point me in the right direction.

Thank you...

**Comment by ceejayemm** [comment]
rAudio-1 is generally a headless application.  Once your RPi is booted you should connect to it from a web browser using either http://rAudio-1.local/ or the ip number of your RPi.  This will open the web pages you use to configure / update rAudio-1.  The current 'base' image of rAudio-1 is i22021117 but you should be able to click on the Cog wheel the the top right hand corner and select Addons.  Form there you will have the option to update to the current latest version which is 20230117.  All other setup/config is done via these web pages, there is very little, if anything, you need to access the RPi command line to do. BTW the first boot of a new SD card will take a bit longer due to rAudio expanding the boot image to use the full size of the new SD card so be a little patient.

Chris
